app/mailers/notify_import_mailer.rb

```diff
@@ -12,27 +12,12 @@ def notify_fail(error_message)
   def notify_success(report)
     set_template('7858c6b6-774e-47f5-80c2-bea221805bb7')
     set_personalisation(
-      environment: app_env,
+      environment: ENV['RAILS_ENV'],
       total_questions: report[:total],
       questions_created: report[:created],
       questions_updated: report[:updated]
     )
     set_email_reply_to(Settings.tech_support_email)
     mail(to: Settings.mail_tech_support)
   end
-
-  private
-
-  def app_env
-    case ENV['SENDING_HOST']
-    when 'trackparliamentaryquestions.service.gov.uk'
-      'production'
-    when 'parliamentary-questions-staging.apps.live-1.cloud-platform.service.justice.gov.uk'
-      'staging'
-    when 'development.trackparliamentaryquestions.service.gov.uk'
-      'dev'
```

Comment:
Everything looks great - good to use the environment variable to set behaviour rather than switching on the domain name. The only wrinkle I can think of is, with Correspondence Tool Mo was thorough in changing all instances of `dev` to `development` and I'm wondering if it's worth doing a similar thing on PQ

The other thing I can't tell is what implications there could be from passing an unexpected value through to the `set_personalisation` method - do you know what that method does or where it comes from? Had a quick search of the app and the web and no joy